src/examples/Logic_fastaSearch.py

```diff
@@ -0,0 +1,225 @@
+                        counter2 = 0
+                        nextLine = next(file_content)
+                        while not nextLine.startswith('>'):
+                            stringValue += nextLine[:-1]
+                            try:
+                                nextLine = next(file_content)
+                            except:
+                                break
+                            counter2 = counter2 + 1
+                        counter = counter + len(stringValue) + len(seqs) + counter2
+                        file_content.seek(counter, 0)
+                        dictKeyAccession[key] = stringValue
+                        proteinList.append(stringValue)
+                        proteinNameList.append(name)
+                        if (os_upper_index == -1):
+                            proteinOSList.append(os)
+                        else:
+                            proteinOSList.append((seqs[descr_upper_index+3:os_upper_index]))
+                    else:
+                        key = seqs.split("OS=")[0]
+                        descr_upper_index = seqs.find('OS=')
+                        # herausfinden bis zu welchem index das OS geht
+                        os_upper_index = seqs.find('(')
+                        os = ''
+                        if (os_upper_index == -1):
+                            zwischenergebnis = seqs[descr_upper_index+3:]
+                            zwischenergebnis1 = zwischenergebnis.split()
+                            os = zwischenergebnis1[0] + ' ' + zwischenergebnis1[1]
+                        name = seqs.split("OS=")[0]  # (seqs[bounds[1]+1:descr_upper_index])
+                        stringValue = ""
+                        counter2 = 0
+                        nextLine = next(file_content)
+                        while not nextLine.startswith('>'):
+                            stringValue += nextLine[:-1]
+                            try:
+                                nextLine = next(file_content)
+                            except:
+                                break
+                            counter2 = counter2 + 1
+                        counter = counter + len(stringValue) + len(seqs) + counter2
+                        file_content.seek(counter, 0)
+                        dictKeyAccession[key] = stringValue
+                        proteinList.append(stringValue)
+                        proteinNameList.append(name)
+                        if (os_upper_index == -1):
+                            proteinOSList.append(os)
+                        else:
+                            proteinOSList.append((seqs[descr_upper_index+3:os_upper_index]))
+        return dictKeyAccession, proteinList, proteinNameList, proteinOSList, dictKeyAccessionDECOY, proteinListDECOY, proteinNameListDECOY, proteinOSListDECOY
+
+    # um im Internet das Protein aufzurufen für mehr Informationen
+
+    def moreProteinInformation(protein_accession):
```

Comment:
more telling name would be something in the line of:
openProteinInformationOnUniprotOrg